web/src/app/profile/duck/actions.js

```diff
@@ -14,8 +20,19 @@ const receiveUserInfoFailure = error => ({
   error
 });
 
+const requestTestAlert = () => ({
+  type: REQUEST_TEST_ALERT
+});
+
+const receiveTestAlert = (success) => ({
+  type: RECEIVE_TEST_ALERT,
+  success
+});
+
 export {
   requestUserInfo,
   receiveUserInfoSuccess,
-  receiveUserInfoFailure
+  receiveUserInfoFailure,
+  requestTestAlert,
+  receiveTestAlert
```

Comment:
Lets rename all the 'test' variables (Only called out here but applies to the commit as a whole)

I get this is test stuff. But since 'test' is included in so many variable names we'll have a good amount of stuff to either remove or rename. I think in general its best to treat this code as 'final' wherever possible and keep the 'test' out of our system.

Reply:
This isn't testing code. This is functionality for the "test alert" button.